files/image_config/hostcfgd/hostcfgd

```diff
@@ -55,12 +56,12 @@ def update_feature_state(feature_name, state):
                 syslog.syslog(syslog.LOG_ERR, "'{}' failed. RC: {}, output: {}"
                               .format(err.cmd, err.returncode, err.output))
                 continue
-        syslog.syslog(syslog.LOG_INFO, "Feature '{}' is enabled and started".format(feature_name))
+        syslog.syslog(syslog.LOG_INFO, "Feature '{}.{}' is enabled and started".format(feature_name, feature_suffix))
     elif state == "disabled":
         stop_cmds = []
-        stop_cmds.append("sudo systemctl stop {}.service".format(feature_name))
-        stop_cmds.append("sudo systemctl disable {}.service".format(feature_name))
-        stop_cmds.append("sudo systemctl mask {}.service".format(feature_name))
+        stop_cmds.append("sudo systemctl stop {}.{}".format(feature_name, feature_suffix))
```

Comment:
When stopping the service, if the service has a .timer file, I believe we need to stop both the timer AND the service. If the timer has already started the service, we need to stop the service. If the timer is currently running and hasn't started the service, we need to stop the time. Thus, we should always stop both.